base/libgit2/oid.jl

```diff
@@ -128,6 +154,11 @@ end
 Base.hex(id::GitHash) = join([hex(i,2) for i in id.val])
 Base.hex(id::GitShortHash) = hex(id.hash)[1:id.len]
 
+"""
+    raw(id::GitHash) -> Vector{UInt8}
+
+Obtain the raw bytes of the [`GitHash`](@ref) as a vector of length $OID_RAWSZ.
```

Comment:
Should this have backticks around OID_RAWSZ?

Reply:
No, it's intended to interpolate into the docstring, since AFAICT `OID_RAWSZ` isn't intended to be user-facing. We do this in a few other places in this file as well.

Reply:
Nice